assets/css/src/components/_welcomeWizard.scss

```diff
@@ -32,6 +32,12 @@
   }
 }
 
+.mailpoet_welcome_wizard_step_revenue_tracking {
+  input[type="submit"] {
```

Comment:
Style lint is already in master and I think this rule should use single quotes https://stylelint.io/user-guide/rules/string-quotes/ so please rebase on master.